### DISCUSSION TITLE
Syntax Error in NR

### DISCUSSION THREAD

**Discussion by valbuz** [pixelit-project/PixelIt#89] [General]
Hallo
Seit kurzem startet bei mir NodeRed immer mal wieder....
Im Log fand ich dies:

![NodeRed_Pixelit_Error](https://user-images.githubusercontent.com/35503440/143842975-29552629-a931-437d-87b3-ee3d4211d329.PNG)

Kennt schon jemand dieses Problem?
Pat

**Comment by o0shojo0o** [comment]
Moin, 

welche Version  hast du den von `node-red-contrib-pixelit` im Einsatz?

**Reply by valbuz** [reply]
0.3.9

**Comment by o0shojo0o** [comment]
Ok, das sieht so aus als ob da irgendwie misst im dein Screen steht den Du das senden möchtest.
Weil an der Stelle hole ich mir den Screen aus dem Cache.
(https://github.com/o0shojo0o/node-red-contrib-pixelit/blob/ce17e5869beff90475be93cacb17a438d9cd4183/pixelit/core.js#L219)

Kannst Du eingrenzen welcher Screen das ist?

**Reply by valbuz** [reply]
Muss bei Gelegenheit mal checken...melde mich dann wieder.
Danke Dir im voraus!

**Reply by valbuz** [reply]
Habe mal einiges deaktiviert. Danach keine Abstürze mehr.
Dann einzelne wieder aktiviert und nun läuft zurzeit alles wieder.
Kann also (noch) nicht sagen, welcher Datenpunkt ztw. keine Daten liefert und den Absturz verursacht...
Jedoch sollte doch NR eine Funktion haben um nicht gleich komplett zu crashen....

**Reply by o0shojo0o** [reply]
An fehlenden Daten von Datenpunkt sollte nicht das Problem sein, das habe ich im Code abgefangen. 
Aber ich kann halt nicht jeden User Fehler voraussehen um diesen im voraus abzufangen. 
Ich werde was einbauen um den Fehler abzufangen, was aber das Grundproblem nicht behebt.

**Reply by o0shojo0o** [reply]
Ich habe ein Issues erstellt. 
https://github.com/o0shojo0o/node-red-contrib-pixelit/issues/11

**Reply by valbuz** [reply]
Läuft bis jetzt :-)
